lib/consumer/data_updater.ex

```diff
@@ -14,7 +15,7 @@ defmodule Mississippi.Consumer.DataUpdater do
   require Logger
 
   # TODO make this configurable?
-  @data_updater_deactivation_interval_ms 10_000
+  @data_updater_deactivation_interval_ms 60 * 60 * 1_000 * 3
```

Comment:
with complicated expressions such as this one, I think it's better to use the timer module for legibility

```suggestion
  @data_updater_deactivation_interval_ms :timer.hours(3)
```

Reply:
Good point! However, this will will be removed once we redesign the message tracking mechanism, so I'd leave it as is for now

Reply:
(also, I would like to make it configurable sooner or later)